src/utils/CrossChainBridge.ts

```diff
@@ -160,11 +180,14 @@ export const handleFederatorVoted = (event: VotedEvent, transaction: Transaction
   // TODO: if token is native to RSK, then symbol should be from token entity and not side token
   crossTransfer.symbol = event.params.symbol
   crossTransfer.sender = event.params.sender
-  crossTransfer.votes = crossTransfer.votes + 1
+  if (isSignatureFederation(event.address.toHex().toLowerCase())) {
+    crossTransfer.votes = crossTransfer.votes = 3
+    crossTransfer.isSigned = true
```

Comment:
This would be much cleaner if Signed was added to the status enum

Reply:
signed has a different meaning, it means all federators have signed the transfer (off-chain) and then one federator executes the transfer in the name of all 3 (so votes jump to 3 and status is changed to executed)